### DISCUSSION TITLE
Git API returns null in the author field.

### DISCUSSION THREAD

**Discussion by pl-alizarch** [community/community#35670] [Apps, API and Webhooks]
I just want to get my public repo commits(commits I'm the author of) via Git API calls. But when i hit the repo endpoint it returns the author as null even i'm the author of those commits. 

Here is the link of my public repo where it is returning the commits(including author as null): 
https://api.github.com/repos/alizarch/faceRecognition/commits

If i gave author param in URL then it returns only commits where the author name is not null like this. 
https://api.github.com/repos/alizarch/faceRecognition/commits?author=alizarch

Why Git is returning author as null even i'm the author and i make commits myself?

**Comment by DSDmark** [comment]
I'm not sure, but it could be help you 

[Click ON](https://github.com/octokit/octokit.net/issues/1433)

And also I suggest you if you want to use author then use like this  ```commit.committer.author.name``` 
I think that is the right form.
